static/certsplainer.js

```diff
@@ -0,0 +1,353 @@
+/*jshint esnext: true */
+window.onload = function() {
+    document.form.addEventListener("change", readfile, false);
+    document.form.addEventListener("submit", send, false);
+    var logs = document.getElementById("logs");
+	let certid = getParameterByName('id');
+	let certsha256 = getParameterByName('sha256');
+	if (certid || certsha256) {
+		loadCert(certid, certsha256);
+	}
+};
+
+function getParameterByName(name, url) {
+    if (!url) {
+      url = window.location.href;
+    }
+    name = name.replace(/[\[\]]/g, "\\$&");
+    var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
+        results = regex.exec(url);
+    if (!results) return null;
```

Comment:
Probably want to wrap the return on this line and the following line in parens to prevent automatic semicolon insertion errors if the code changes in the future.